src/stories/material/Material.tsx

```diff
@@ -1,28 +1,35 @@
+import clsx from "clsx";
+
 export type MaterialProps = {
   url: string;
   animate: boolean;
   size: "small" | "medium" | "large";
+  tint: "20" | "40" | "60" | "80" | "120";
 };
 
-const placeholderBg = [
-  "bg-identity-tint-120",
-  "bg-identity-tint-80",
-  "bg-identity-tint-60",
-  "bg-identity-tint-40",
-  "bg-identity-tint-20",
-];
-
 export const Material = (props: MaterialProps) => {
-  // If Material does not have an img, show a random bg-color for each card.
-  const bg = placeholderBg[(Math.random() * placeholderBg.length) | 0];
+  const { size, animate, url } = props;
+
+  const classes = {
+    wrapper: clsx(
+      `material material--${size}`,
+      { "bg-identity-tint-120": props.tint === "120" },
+      { "bg-identity-tint-80": props.tint === "80" },
+      { "bg-identity-tint-60": props.tint === "60" },
+      { "bg-identity-tint-40": props.tint === "40" },
+      { "bg-identity-tint-20": props.tint === "20" },
+      { material__animate: animate }
+    ),
+  };
+
   return (
     <div className="material-container">
-      <a
-        className={`material material--${props.size} ${bg} ${
-          props.animate ? "material__animate" : ""
-        }`}
-      >
-        <img src={props.url} alt="" onError={(e) => (e.currentTarget.style["display"] = "none")} />
+      <a className={classes.wrapper}>
+        <img
+          src={url}
+          alt=""
+          onError={(e) => (e.currentTarget.style["display"] = "none")}
```

Comment:
I was not aware of this construction the first time I reviewed.
Just curious: What is happening here? If an image cannot be displayed it gets `style="display: none"` and is hidden?

Reply:
This relates to work done by @PELN in [57920b9](https://github.com/reload/dpl-design-system/commit/57920b9350931f8ce1d1c94524b9f7a06e0c7e8f)

She writes: "Add a function to handle broken image icon which should not be displayed"

Perhaps she can provide more details. It is not related to the current PR anyway.

Reply:
I am resolving this, as it is not related to the PR.